src/compiler/propagation/type_scope.cc

```diff
@@ -21,6 +21,8 @@
 namespace toit {
 namespace compiler {
 
+static const int EXTRA = 1;
```

Comment:
Add a comment, what this constant is for.

Reply:
Done.